content/concepts/basics/03-content-relations.md

```diff
@@ -4,17 +4,68 @@ metaTitle: "sensenet - Content Relations"
+This structure lets you organize, group your content easily by creating separate subtrees. It's not only important because of the possibility to configure content accessibility for different types of users on different parts of the tree, but also because of the inheritance of several things like content types' underlying business logic, metadata field configurations and settings.
+
+# Parent-child relation
+Content can inherit multiple things from its ancestors, from fields to permission the possibilities are basically infinite.
+
+## Permission inheritance
+Set a permission on a container item on a higher level (e.g. a document library or a workspace) it will be inherited by its children. This way you do not have to set permissions on every subfolder or document, because all content inherits permissions from their parent.
+
+## Settings inheritance
+In sensenet, settings are stored as content in the content repository so you can take all the advantages of the tree structure in case of settings as well.
+
+## Content type inheritance
+Content types are defined in a type hierarchy. A content type may be inherited from another type automatically inheriting its fields and the underlying business logic (handler). 
+
+> ([learn more about content tree and inheritance](/concepts/basics/02-content-tree))
+
+# Reference field (links)
+
+The reference field is used for defining references to other content(s). When a content holds references pointing to other content (for example a group referencing users) it is done with a reference field.
+
+The following apply to the behavior of the field:
+
+- **number of references:** a reference field can be configured to hold either one or more references
+- **default reference:** one or more default references can be set via field configuration
+- **types of referable content:** types of referable content can be restricted
+- **location of referable content:** location of referable content can be restricted
+- **set of referable content:** set of referable content can be defined with an optional query
+- **moving/renaming references:** when a reference is moved to another place or it has been renamed, it does not affect the content it is referenced by - the moved/renamed reference is still visible in the feference field
+- **permission handling:** if the current user does not have see permissions to one of the references, that reference is not visible for the user
+- **copying content with references:** when a content with a reference field is copied, the newly created content will hold the same references as the source content. 
+
+(Maybe it's not that important here, could be deleted:
+Copying source content references along with the source content does not affect this behavior, the newly created content will hold references to the originally referenced content and not the newly created copies.)
+
+## Configuration
+The following properties can be set in the field's ``FieldSetting`` configuration:
```

Comment:
```suggestion
When configuring the reference field you can define whether multiple references are allowed or only a single content can be referenced. You can restrict referenced content by content type or location (in the tree). Default referenced content(s) can also be defined so when a new content is created it will automatically have a predefined set of content in the reference field.
```